styles.css

```
@@ -0,0 +1,267 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+html {
+  font-size: 16px;
+}
+
+body {
+  font-family: "Poppins";
+  margin: 0;
+
+  background-color: #f8f4f4;
+}
+
+/* #app {
+  max-width: 1440px;
+  margin: 0 auto;
+} */
+
+.d-flex {
+  display: flex;
+}
+.center {
+  justify-content: center;
+}
+.separator {
+  border-top: 1px solid rgba(0, 0, 0, 0.1);
+}
+#main-header {
+  box-shadow: 0 4px 13px -3px rgb(0 0 0 / 10%);
+  border-bottom: 1px solid #d2d2d2;
+  padding: 10px 15px;
+}
+
+#header-top {
+  flex-wrap: wrap;
+  max-width: 1440px;
+  margin: 0 auto;
+  display: flex;
+  flex: 1;
+  justify-content: space-between;
+}
+#npm-text {
+  padding-left: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-grow: 1;
+  font-size: 14px;
+}
+
+#main-nav ul {
+  margin: 0;
+  padding-left: 0;
+  flex-wrap: wrap;
+  display: flex;
+}
+
+#main-nav ul li {
+  list-style: none;
+  padding: 10px;
+}
+
+#main-nav ul li a {
+  text-decoration: none;
+  color: black;
+}
+
+#npm-logo {
+  margin-right: 30px;
+  width: 70px;
+  align-self: center;
+}
+#hearth {
+  width: 30px;
+  align-self: center;
+}
+
+#glass {
+  width: 20px;
+  padding: 10px 5px 10px 15px;
+  align-self: center;
+}
+
+#header-bottom {
+  flex-wrap: wrap;
+  padding: 10px 0;
+  max-width: 1440px;
+  margin: 0 auto;
+  display: flex;
+  flex: 1;
+}
+
+#form-container {
+  display: flex;
+  flex-grow: 1;
+}
+#text-input {
+  padding: 5px;
+  background-color: #f2f2f2;
+  border: 2px solid transparent;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-grow: 1;
+}
+#text-input input {
+  background: none;
+  font-size: 16px;
+  padding-left: 10px;
+  border: unset;
+  height: 40px;
+  outline: none;
+  width: calc(100% - 10px);
+}
+
+#form-container button {
+  cursor: pointer;
+  padding: 8px 32px;
+  color: #fff;
+  border-style: none;
+  border-width: 0;
+  font-weight: bold;
+  background-color: #fb3b49;
+}
+
+#text-input:focus-within {
+  border-color: #fb3b49;
+}
+
+#btn-group-session {
+  padding: 5px 0;
+  display: flex;
+  margin-left: 20px;
+}
+
+.btn-default {
+  text-decoration: unset;
+  padding: 13px 28px;
+  font-size: 14px;
+  border: unset;
+  background: none;
+  color: black;
+}
+
+.btn-large {
+  text-decoration: unset;
+  padding: 20px;
+  min-width: 250px;
+  text-align: center;
+  color: black;
+  font-weight: 700;
+}
+
```

Agregaste el id para este botón pero no tiene color estaría bien agregarlo a ese id que ya tienes 👍 
```suggestion
#join {
background-color: white;
}
```